src/sidebar/services/groups.ts

```diff
@@ -464,6 +468,15 @@ export class GroupsService {
       // Persist this group as the last focused group default
       this._store.setDefault('focusedGroup', newGroupId);
     }
+
+    if (
+      // For now, group members are going to be used for at-mentions only, so we
+      // can skip loading them if the feature is not enabled
```

Comment:
The feature flag is just a temporary measure in development. We plan to ship a version of this where the flag is enabled for everyone. At that point we'll be fetching group members regardless of whether the user edits any annotations in the session.